src/CMakeLists.txt

```
@@ -31,7 +43,15 @@ install(TARGETS simdjson-source EXPORT simdjson-config)
 
 if(SIMDJSON_BUILD_STATIC)
   MESSAGE( STATUS "Building a static library." )
-  add_library(simdjson STATIC "")
+  ###
+  # In the case of a static library, if you do "target_link_libraries(simdjson PRIVATE simdjson-source simdjson-internal-flags)"
+  # without also calling "install(TARGETS simdjson-source EXPORT simdjson-config)" you get an error
+  # to the effect that you are trying to install 'simdjson' which depends on 'simdjson-source' while
+  # not installing 'simdjson-source', so it fails. So we bypass entirely simdjson-source since we do
+  # not want to install them.
+  ####
+  add_library(simdjson STATIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>/simdjson.cpp)
```

```suggestion
  add_library(simdjson STATIC simdjson.cpp)
```
should be enough. You can also extract this file from the target by using 
`get_target_property(SIMDJSON_SOURCES simdjson SOURCES)`